ingestion/src/metadata/ingestion/ometa/mixins/user_mixin.py

```diff
@@ -39,6 +39,29 @@ class OMetaUserMixin:
         + ES_INDEX_MAP[User.__name__]
     )
 
+    # Allow for more flexible name lookup
+    name_search = (
+        "/search/query?q={name}&from={from_}&size={size}&index="
+        + ES_INDEX_MAP[User.__name__]
```

Comment:
we should look up both user and team right?

Reply:
adding it